test/Quoter.t.sol

```diff
@@ -480,33 +362,19 @@ contract QuoterTest is Test, Deployers, GasSnapshot {
 
         IQuoter.QuoteExactParams memory params = getExactOutputParams(tokenPath, 6223);
 
-        (
-            int128[] memory deltaAmounts,
-            uint160[] memory sqrtPriceX96AfterList,
-            uint32[] memory initializedTicksLoadedList
-        ) = quoter.quoteExactOutput(params);
+        (uint256 amountIn,) = quoter.quoteExactOutput(params);
 
-        assertEq(deltaAmounts[0], 6283);
-        assertEq(sqrtPriceX96AfterList[0], 79708304437530892332449657932);
-        assertEq(initializedTicksLoadedList.length, 1);
-        assertEq(initializedTicksLoadedList[0], 2);
+        assertEq(amountIn, 6283);
```

Comment:
assert gas is > 0? I know it's hard to get accurate on the gas but we know it should at least not be 0

Reply:
I've done asserts `50k < gas < 400k`